modules/virt-vm-wizard-fields-web.adoc

```diff
@@ -84,9 +84,16 @@ endif::[]
 
 |Flavor
 |Tiny, Small, Medium, Large, Custom
-|Presets that determine the amount of CPU and memory allocated to the virtual machine. The presets displayed for *Flavor* are determined by the operating system.
+|An annotation in a virtual machine template with predefined values that determine the amount of CPU and memory allocated to the virtual machine, depending on the operating system associated with that template.
```

Comment:
I'd not change the beginning of the sentence that muhc, thus, replace "An annotation in a virtual machine template with predefined values that determine the amount" with "Presets the amount …"

Reply:
I've reworded based on your suggestion. What do you think?